src/command.coffee

```diff
@@ -39,6 +39,7 @@ SWITCHES = [
   ['-i', '--interactive',     'run an interactive CoffeeScript REPL']
   ['-j', '--join [FILE]',     'concatenate the source CoffeeScript before compiling']
   ['-m', '--map',             'generate source map and save as .js.map files']
+  ['-I', '--inline',          'store sourcemap directly in compiled .js file']
```

Comment:
@michaelficarra thanks for your prompt review.

I've switched to your recommended commands and also ensured that `--map` is enabled by default with `--inline-map`.

Also I cleaned up URI generation a little. In `browser.coffee` it was encoding and decoding the uri for no reason (that I know of). I looked at how [babel is doing it](https://github.com/thlorenz/convert-source-map/blob/master/index.js#L65) and they don't do this.

Again, let me know if anything else is needed.